site2/docs/io-quickstart.md

```diff
@@ -378,11 +338,415 @@ cqlsh:pulsar_test_keyspace> select * from pulsar_test_table;
+    Version: '5.7.26'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
+    ```
+
+4. Access to MySQL.
+
+    ```
+    $ docker exec -it pulsar-mysql /bin/bash
+    mysql -h localhost -uroot -pjdbc
+    ```
+
+5. Create a _pulsar_mysql_jdbc_sink_ table.
+
+    ```
+    $ create database pulsar_mysql_jdbc_sink;
+
+    $ use pulsar_mysql_jdbc_sink;
+
+    $ create table if not exists pulsar_mysql_jdbc_sink
+    (
+    id INT AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    primary key (id)
+    )
+    engine=innodb;
+    ```
+
+## Configure a JDBC sink
+
+Now that we have a MySQL running locally. In this section, we will configure a JDBC sink connector. The JDBC sink connector will read messages from a Pulsar topic and write messages into a MySQL table.
+
+1. Add a configuration file.   
+   
+    To run a JDBC sink connector, you need to prepare a yaml config file including the information that Pulsar IO runtime needs to know. For example, how Pulsar IO can find the MySQL cluster, what is the JDBCURL and the table that Pulsar IO will use for writing messages to.
+
+    Create a _pulsar-mysql-jdbc-sink.yaml_ file , copy the following contents to this file, and place the file in the `pulsar/connectors` folder.
+
+    ```
+    configs:
+      userName: "root"
+      password: "jdbc"
+      jdbcUrl: "jdbc:mysql://127.0.0.1:3306/pulsar_mysql_jdbc_sink"
+      tableName: "pulsar_mysql_jdbc_sink"
+    ```
+
+2. Create a schema.
+
+    Create a _avro-schema_ file, copy the following contents to this file, and place the file in the `pulsar/connectors` folder.
+
+    ```
+    {
+      "type": "AVRO",
+      "schema": "{\"type\":\"record\",\"name\":\"Test\",\"fields\":[{\"name\":\"id\",\"type\":[\"null\",\"int\"]},{\"name\":\"name\",\"type\":[\"null\",\"string\"]}]}",
+      "properties": {}
+    }
+    ```
+
+    > #### Tip
+    >
+    > For more information about AVRO, see [Apache Avro Documentation](https://avro.apache.org/docs/1.8.2/).
+
+
+3. Upload a schema to a topic.  
+
+    This example uploads the _avro-schema_ schema to the _pulsar-mysql-jdbc-sink-topic_ topic.
+
+    ```
+    $ bin/pulsar-admin schemas upload pulsar-mysql-jdbc-sink-topic -f ./connectors/avro-schema
+    ```
+
+4. Check if the schema has been uploaded successfully.
+
+    ```
+    $ bin/pulsar-admin schemas get pulsar-mysql-jdbc-sink-topic
+    ```
+
+    The schema has been uploaded successfully if the following message appears.
+
+    ```
+    {
+      "name" : "pulsar-mysql-jdbc-sink-topic",
+      "schema" : "eyJ0eXBlIjoicmVjb3JkIiwibmFtZSI6IlRlc3QiLCJmaWVsZHMiOlt7Im5hbWUiOiJpZCIsInR5cGUiOlsibnVsbCIsImludCJdfSx7Im5hbWUiOiJuYW1lIiwidHlwZSI6WyJudWxsIiwic3RyaW5nIl19XX0=",
+      "type" : "AVRO",
+      "properties" : { }
+    }
+    ```
+
+## Submit a JDBC sink
+
+Pulsar provides the [CLI](https://pulsar.apache.org/docs/en/pulsar-admin/) for running and managing Pulsar I/O connectors.
```

Comment:
`[CLI](admin-api-overview.md)`

Reply:
Thanks for reminding, I've changed.

Here I want to show all the contents on the CLI page, URL might be changed, so referencing a `.md` file is easier for maintenance.